instat/dlgAddLink.vb

```diff
@@ -35,28 +37,108 @@ Public Class dlgAddLink
 
     Private Sub InitialiseDialog()
         ucrBase.iHelpTopicID = 506
+        ucrDataSelectorFrom.SetParameter(New RParameter("from_data_frame", 0))
+        ucrDataSelectorFrom.SetParameterIsString()
+        ucrDataSelectorTo.SetParameter(New RParameter("to_data_frame", 0))
+        ucrDataSelectorTo.SetParameterIsString()
+        ucrInputLinkName.SetParameter(New RParameter("link_name", 0))

```

Comment:
Why does `link_name` and `to_data_frame` parameters have the same parameter position yet they are in the same function?